src/middleware/ErrorMiddleware.js

```diff
@@ -34,29 +28,28 @@ module.exports = function (
       res.format({
         text: () => this.sendTextResponse(err, req, res),
         html: () => this.sendHtmlResponse(err, req, res),
-        json: () => this.sendJsonResponse(err, req, res)
+        json: () => this.sendJsonResponse(err, req, res),
       });
 
       next();
     }
 
     logErrorStack(err) {
-      const statusCode = err.statusCode || 0;
```

Comment:
Assigning fallback `statusCode` of `0` is not necessary, it's only use to check it the status code exists in  `EXCLUDE_STATUSCODE_FROM_LOGS`, which doesn't contains `null` or `undefined`.

According to the unit test coverage, the `|| 0` "branch" is never executed

Reply:
I don't follow, when using the middleware as expected, `this.EXCLUDE_STATUSCODE_FROM_LOGS` will always be `[404]`, and calling `some` on this array with an `undefined` value is the same as calling it with `0`, i.e. it returns `false`.
![image](https://github.com/opentable/spur-web/assets/118707423/31f52621-a7b0-4c5e-9574-bea6cdf7238a)

When calling the method directly, e.g. `ErrorMiddleware.logErrorState({statusCode: 404})`, `this.EXCLUDE_STATUSCODE_FROM_LOGS` is `undefined` and will _always_ return `false`, which obviously is not expected. 

I will revert the change, but it defies logic